docs/getting-started/tutorial-load-dataset.asciidoc

```diff
@@ -151,15 +151,21 @@ The accounts data set doesn't require any mappings, so at this point we're ready
 {ref}/docs-bulk.html[`bulk`] API to load the data sets with the following commands:
 
 [source,shell]
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/bank/account/_bulk?pretty' --data-binary @accounts.json
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/shakespeare/doc/_bulk?pretty' --data-binary @shakespeare_6.0.json
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/_bulk?pretty' --data-binary @logs.jsonl
+<host> Elasticsearch listens at localhost by default
+<port> Elasticsearch HTTP Port defaults to 9200
+curl -u elastic -H 'Content-Type: application/x-ndjson' -XPOST '<host>:<port>/<index-name>/<type>/_bulk?pretty' --data-binary @<filename>
```

Comment:
This should be removed as it's the template of the three commends which need to be run and not apparent what you should do with it.

Reply:
I would recommend that we include the command template in the text above the sample. The sample commands, as-is, do not work for Cloud.